src/Controller/EventsController.php

```diff
@@ -747,7 +747,13 @@ public function approveAction(Request $request, PDO $db): void
         }
 
         // Send a notification email as we have approved
-        $event        = $event_mapper->getEventById($event_id, true)['events'][0];
+        $event        = $event_mapper->getEventById($event_id, true);
+        if ($event === false) {
```

Comment:
Can we switch getEventById() to throw rather than sometimes return false? I realize that'll change a fair number of places, but that should clean things up nicely, and we can throw a 404 right from the mapper.

Reply:
probably resolved - I've gone through the usages a couple of times and I think cleaned up all 24 but double-checking is appreciated.